recipes/libaec/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "libaec" %}
+{% set version = "v1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.dkrz.de/k202009/{{ name }}/-/archive/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c1aa0d6d8c070f5b1d65a42bc64334fbd69518ea38ab566fb52880bae6f1a72e
+
+build:
+  number: 0
+  skip: true  # [win]
```

Comment:
How about adding a run_exports here?

Reply:
Not sure, what I should add there...

Reply:
Something like,
```
build:
   run_exports:
     - {{ pin_subpackage("libaec", max_pin="x.x") }}
```

"x.x" means if a downstream package is built with libaec v.1.0.2, it will be pinned as `>=1.0.2,<1.1` at runtime. You'll have to decide whether to use "x.x" or "x" (default) or "x.x.x"